app/model/Event/Factory/EducationFactory.php

```diff
@@ -27,6 +28,7 @@ public function create(stdClass $skautisEducation): Education
             $skautisEducation->EndDate === null ? null : Date::createFromFormat(self::DATETIME_FORMAT, $skautisEducation->EndDate),
             $skautisEducation->Location ?? '',
             $skautisEducation->ID_EventEducationState ?? '',
+            new SkautisGrantId($skautisEducation->ID_Grant ?? null),
```

Comment:
kdyz $skautisEducation->ID_Grant je Null, tak pak posleš do konstruktoru zase null i kdyz to smí být jen int

Reply:
Good catch, díky. Neměl by tohle chytit phpstan?

Reply:
taky bych čekal, že to chytne